releasenotes/notes/apm-disable-file-logging-3cd11cf32b25b36d.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    APM: The `disable_file_logging` setting is now respected.
```

Comment:
It could possibly be considered a bug fix or an enhancement, but I think the "features" section usually refers to bigger changes or new products?

Reply:
Ah yeah true. Changed to enhancements (based on https://github.com/DataDog/datadog-agent/pull/13613/files#diff-6d2be1202f21a64f45cfc79ed174869fae3fd48189c3a2ca9c721173d6882755)